mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -50,6 +52,15 @@ interface MavericksView : LifecycleOwner {
             )
         }
 
+    val mavericksViewSubscriptionViewModel: MavericksViewSubscriptionViewModel
+        get() = when (this) {
+            is ViewModelStoreOwner -> ViewModelProvider(this).get(MavericksViewSubscriptionViewModel::class.java)
```

Comment:
might as well combine the MavericksViewIdViewModel and this one, into something like `MavericksViewInternalViewModel` and use it for all internal data we want to manage